### ISSUE TITLE
didn;t found files in window

### ISSUE THREAD

**Issue by walkadaw** [wgrabowski/ngx-unused#19]
When run command on window pc get error what ts files not found. 
```
npx ngx-unused src -p src/tsconfig.app.json
```

Main problem than in `globSync`
https://github.com/wgrabowski/ngx-unused/blob/master/lib/createProject.ts#L34

includePatterns =`C:\\project\\src` but globSync can't work with in. it expected `C:/project/src`.

```
Angular CLI: 17.1.0
Node: 20.9.0
Package Manager: npm 10.4.0
OS: win32 x64
```

**Comment by wgrabowski** [comment]
please provide cli params and exact error that was printed

**Comment by walkadaw** [comment]
`No relevant .ts files found in provided source root(s)`

`npx ngx-unused src -p src/tsconfig.app.json`

But .ts files have

**Comment by silk-bahamut** [comment]
same problem here on windows too on a standard app

`npx ngx-unused src/app -p tsconfig.app.json`
gives me 

_No relevant .ts files found in provided source root(s)_

**Comment by wgrabowski** [comment]
Fixed by https://github.com/wgrabowski/ngx-unused/pull/21 and published in version 1.2.0
